website_product_document_download_counter/readme/USAGE.md

```diff
@@ -0,0 +1,13 @@
+1.  Install the module from the Odoo interface.
```

Comment:
This step is obvious and should be omitted. Not to mention, that it is not related to usage.